### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by immint023**

### CODE REVIEW

File: .github/workflows/workflow-config.yml [hunk 1]
```diff
@@ -36,6 +36,8 @@ jobs:
         run: npm ci
       - name: Run Type check
         run: npm run typecheck
+      - name: Run Type check
+        run: npm run typecheck
```
Comment (davidnguyen179) [inline .github/workflows/workflow-config.yml:40]: why is it duplicated?

Comment (immint023) [reply .github/workflows/workflow-config.yml]: I have fixed it.

File: package.json [hunk 2]
```diff
@@ -44,8 +45,12 @@
     "@types/chrome": "0.0.129",
     "@types/jest": "26.0.20",
     "@types/sinon": "9.0.10",
+    "@typescript-eslint/eslint-plugin": "^4.17.0",
```
Comment (davidnguyen179) [inline package.json:48]: can you use fixed version? remove `^`

Comment (immint023) [reply package.json:48]: how about package-lock.json file ?

Comment (davidnguyen179) [reply package.json:48]: I guess you have to `uninstall` & `install` again at ur local machine.

File: package.json [hunk 3]
```diff
@@ -12,7 +12,8 @@
     "firefox": "rimraf dist && env BROWSER='firefox' webpack --config webpack/webpack.prod.js",
     "edge": "rimraf dist && env BROWSER='edge' webpack --config webpack/webpack.prod.js",
     "typecheck": "./node_modules/.bin/tsc --noEmit",
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint src/**"
```
Comment (davidnguyen179) [inline package.json:16]: I think in `ci`. We have to add `lint` command to check, right?

Comment (immint023) [reply package.json:16]: I thought we will check lint at local before pushing code to remote so checking lint at ci is redundant.

Comment (davidnguyen179) [reply package.json:16]: we can completely bypass githook by `git push --no-verify`. So it's better to check in CI as well

File: src/lib/spotify.ts [hunk 4]
```diff
@@ -206,30 +188,22 @@ export async function removeTrack(songInfo: TrackInfo, accessToken: string) {
...
-    return await fetch(url, {
-      method: 'PUT',
-      headers: {
-        Authorization: `Bearer ${accessToken}`,
-      },
-    });
-  } catch (e) {
-    throw e;
-  }
+  return await fetch(url, {
```
Comment (davidnguyen179) [inline src/lib/spotify.ts:203]: So, eslint automatically formats it, doesn't it?

Comment (immint023) [reply src/lib/spotify.ts:203]: no, i have changed it.

File: package.json (hunk 3 above)
Comment (davidnguyen179) [reply package.json]: did you add to github actions?

**Comment by davidnguyen179** [comment]
Looks nice!

**Pull request merged**